yt_dlp/extractor/xvideos.py

```diff
@@ -90,6 +90,51 @@ class XVideosIE(InfoExtractor):
     }, {
         'url': 'https://de.xvideos.com/video4588838/biker_takes_his_girl',
         'only_matching': True
+    }, {
+        'url': 'https://flashservice.xvideos.com/embedframe/ucuvbkfda4e',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.xvideos.com/embedframe/ucuvbkfda4e',
+        'only_matching': True,
+    }, {
+        'url': 'http://static-hw.xvideos.com/swf/xv-player.swf?id_video=ucuvbkfda4e',
+        'only_matching': True,
+    }, {
+        'url': 'http://xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'https://xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
```

Comment:
these 2 are dupes, can remove

Reply:
these are not duplicates. http/http**s**

Reply:
the first test highlighted here is an exact dupe of the very first test in the `_TESTS` list, no?

and we don't need to test http vs https, it is convention to always match both http and https in the `_VALID_URL`

Reply:
i remove 103-108 lines.